roles/federation-demo/tasks/postgres.yml

```
@@ -0,0 +1,47 @@
+---
+- name: Make sure psycopg2 is installed
+  command: pip install psycopg2==2.7.5 --ignore-installed
+
+- name: Send file
+  copy:
+    src: private_files/setup_db.sql
+    dest: "{{federation_db.db_script_path}}"
+
+
+- name: Create a new database with name service_registry_db
+  postgresql_db:
+    name: service_registry_db
+  become: yes
+  become_user: postgres
+
+- name: Create PostgreSQL tables for Federation Registry via SQL scripts
+  postgresql_query:
+    db: "{{ federation_db.db_name }}"
+    login_user: "{{ federation_db.db_username }}"
+    login_password: "{{ federation_db.db_password }}"
+    path_to_script: "{{ federation_db.db_script_path }}"
+    login_host: "{{ federation_db.db_host }}"
+  register: postgresql_result
+  failed_when: "postgresql_result is failed and 'already exists' not in postgresql_result.msg"
+  when: "federation_db.db_name is defined"
+
+- name: Send file
```

```suggestion
- name: Send a magic file ^_^
```

Could you have a more representative name/message here please ?